src/frontend/citizen-portal/src/app/shared/dialogs/image-ticket-not-found-dialog/image-ticket-not-found-dialog.component.html

```diff
@@ -29,13 +29,20 @@
       It appears that your ticket is not a Motor Vehicle Act violation ticket.
     </p>
   </div>
-  <div>
+  <div *ngIf="options.messageKey === '!error4'">
     <span class="flex-grow-1"></span>
     <p>Please
       <a href="https://www2.gov.bc.ca/gov/content/justice/courthouse-services/fines-payments/pay-dispute-ticket/prov-violation-tickets/dispute-ticket"
         target="blank">click here</a> for more options.
     </p>
   </div>
+  <div *ngIf="options.messageKey === 'error4'">
```

Comment:
have you checked if you can submit an 'S' ticket that is more than 30 days old and it will show up in the staff portal?

Reply:
yes, it is showing in staff portal when the ticket is more than 30 days for camera ticket